kolibri/core/logger/csv_export.py

```diff
@@ -103,7 +124,37 @@ def cache_content_title(obj):
     )
 )
 
-map_object = partial(output_mapper, labels=labels, output_mappings=mappings)
+
+def get_max_ancestor_depth():
+    max_depth = 0
+    content_ids = ContentSummaryLog.objects.values_list("content_id", flat=True)
+    nodes = (
+        ContentNode.objects.filter(content_id__in=content_ids)
+        .only("content_id")
+        .iterator(chunk_size=BATCH_SIZE)
```

Comment:
I don't know if this will address the issue I was suggesting, which is where the performance issue might come from the length of the `content_ids` subquery - so, maybe let's just remove the iterator for now, and just see whether there are actually performance issues here in practice when we test.